<a href="https://colab.research.google.com/github/HaJeong-K/Jeju_project/blob/main/2020.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## 2020 년
 # 데이터 전처리

 데이터 수집 비율

- 2020년도 제주 여행자를 대상으로 한 방문 관광객 실태조사에서 응답자 프로파일을 가져옴.
응답자 프로파일의 카테고리 중 월별 카테고리만 수집함.

- 해당 연도의 데이터 수집 대상 비율을 명시해 앞으로 다룰 데이터들이 어떤 방식으로 추출되었는지 설명함.

In [ ]:
import pandas as pd
from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


In [ ]:
profile_df_2020 = pd.read_excel('/content/drive/MyDrive/2020 profile.xlsx')
profile_df_2020.head()

table Unnamed: 1 Unnamed: 2
0   NaN        NaN         전체
1   NaN        NaN        사례수
2  [전체]        NaN       3000
3    월별        08월        500
4    월별        09월        500

In [ ]:
columns = {'table' : '분석 카테고리',
           'Unnamed: 1' : '세부 카테고리',
           'Unnamed: 2' : '사례수',
           'Unnamed: 3' : '비율(%)'}

profile_df_2020 = profile_df_2020.rename(columns = columns)
profile_df_2020

분석 카테고리     세부 카테고리   사례수
0      NaN         NaN    전체
1      NaN         NaN   사례수
2     [전체]         NaN  3000
3       월별         08월   500
4       월별         09월   500
5       월별         10월   500
6       월별         11월   500
7       월별         12월   500
8       월별     21년 01월   500
9     거주지역          서울  1269
10    거주지역  경인권(경기/인천)   856
11    거주지역         광역시   529
12    거주지역      그 외 지역   346
13      성별          남자  1265
14      성별          여자  1735
15      연령      20대 이하   504
16      연령         30대  1144
17      연령         40대   825
18      연령      50대 이상   527
19    방문목적   휴가 및 순수여행  2668
20    방문목적      비즈니스여행    67
21    방문목적        기타여행   265

In [ ]:
profile_df_2020 = profile_df_2020.drop(index = [0, 1]).reset_index(drop=True)
profile_df_2020

분석 카테고리     세부 카테고리   사례수
0     [전체]         NaN  3000
1       월별         08월   500
2       월별         09월   500
3       월별         10월   500
4       월별         11월   500
5       월별         12월   500
6       월별     21년 01월   500
7     거주지역          서울  1269
8     거주지역  경인권(경기/인천)   856
9     거주지역         광역시   529
10    거주지역      그 외 지역   346
11      성별          남자  1265
12      성별          여자  1735
13      연령      20대 이하   504
14      연령         30대  1144
15      연령         40대   825
16      연령      50대 이상   527
17    방문목적   휴가 및 순수여행  2668
18    방문목적      비즈니스여행    67
19    방문목적        기타여행   265

In [ ]:
profile_df_2020 = profile_df_2020.drop(index=range(7, 20)).reset_index(drop=True)

profile_df_2020

분석 카테고리  세부 카테고리   사례수
0    [전체]      NaN  3000
1      월별      08월   500
2      월별      09월   500
3      월별      10월   500
4      월별      11월   500
5      월별      12월   500
6      월별  21년 01월   500

## 1인당 지출 경비 전처리
- 1인당 지출 경비에 대한 카테고리: 전체 부분, 숙박 부분, 식음료 부분, 항공/선박 부분, 관광/문화 지출 부분
- 1인당 지출 경비 부분이 지출 기준인원 항목의 하위 항목으로 분류되어 있어 응답자 프로파일처럼 해당 데이터의 비율을 명시함.

In [ ]:
expenditure_standard_2020 = pd.read_excel('/content/drive/MyDrive/2020 지출 인원.xlsx')
expenditure_standard_2020.head()

table Unnamed: 1 Unnamed: 2 Unnamed: 3 Unnamed: 4 Unnamed: 5 Unnamed: 6  \
0   NaN        NaN        사례수         1명         2명         3명         4명   
1  [전체]        NaN       3483       10.6       41.3       17.9       19.8   
2    구분     01-02월        928         11         40       13.2       22.5   
3    구분     08-12월       2555       10.4       41.7       19.7       18.8   
4    월별        01월        602       12.8       37.5       13.4       22.3   

  Unnamed: 7 Unnamed: 8 Unnamed: 9 Unnamed: 10 Unnamed: 11 Unnamed: 12  \
0         5명         6명         7명          8명          9명         10명   
1        5.4        1.7        1.1         0.8         0.2         0.5   
2        5.8        1.9          2           1         0.3         0.8   
3        5.2        1.7        0.7         0.7         0.2         0.3   
4        5.8        1.3        2.5         0.7         0.4         1.2   

  Unnamed: 13 Unnamed: 14  
0      11명 이상          평균  
1         0.7        2.98  
2         1.4        3.17  
3         0.5        2.91  
4           2        3.26

In [ ]:
columns = {'table' : '분석 카테고리',
           'Unnamed: 1' : '세부 카테고리',
           'Unnamed: 2' : '사례수',
           'Unnamed: 3' : '1명',
           'Unnamed: 4' : '2명',
           'Unnamed: 5' : '3명',
           'Unnamed: 6' : '4명',
           'Unnamed: 7' : '5명',
           'Unnamed: 8' : '6명',
           'Unnamed: 9' : '7명',
           'Unnamed: 10' : '8명',
           'Unnamed: 11' : '9명',
           'Unnamed: 12' : '10명',
           'Unnamed: 13' : '11명 이상',
           'Unnamed: 14' : '평균'}

expenditure_standard_2020= expenditure_standard_2020.rename(columns = columns)
expenditure_standard_2020

분석 카테고리       세부 카테고리   사례수    1명    2명    3명    4명    5명   6명   7명  \
0         NaN           NaN   사례수    1명    2명    3명    4명    5명   6명   7명   
1        [전체]           NaN  3483  10.6  41.3  17.9  19.8   5.4  1.7  1.1   
2          구분        01-02월   928    11    40  13.2  22.5   5.8  1.9    2   
3          구분        08-12월  2555  10.4  41.7  19.7  18.8   5.2  1.7  0.7   
4          월별           01월   602  12.8  37.5  13.4  22.3   5.8  1.3  2.5   
5          월별           02월   327   7.7  44.6  12.9  22.9   5.8  2.9  1.2   
6          월별           08월   614   8.9    39  19.6  24.8   4.8    1  0.6   
7          월별           09월   396  14.4  44.4    18  13.5   5.5  2.7  0.8   
8          월별           10월   584    12  40.7  20.9  15.9   5.1  2.4  0.8   
9          월별           11월   620   7.9  43.4  20.7  16.8   6.2  1.8    1   
10         월별           12월   341  10.6  42.3  17.9    23   4.1  0.6  0.4   
11        거주지         서울특별시  1259  12.9  41.1  19.3  18.4   5.8  1.2  0.3   
12        거주지         부산광역시   262  10.8  42.9  14.8  20.8   3.9  2.1  0.4   
13        거주지         대구광역시   139   5.9  49.8  18.3  15.6   2.8  0.6  0.9   
14        거주지         인천광역시   215   9.3  42.4  22.6  15.2   3.9  0.6  2.9   
15        거주지         광주광역시    95  20.5  35.3  11.3  13.5   6.4  1.4  4.9   
16        거주지         대전광역시    76  15.5  41.9  16.8  13.8   8.1  2.4  1.6   
17        거주지         울산광역시    52   1.5  58.9   6.6  24.4   8.5    0    0   
18        거주지         세종자치시    28   4.8  23.5  32.4  13.9  11.9  4.1    0   
19        거주지           경기도   832   8.2    42  19.4  21.1   4.9  2.2  0.7   
20        거주지           강원도    33     6  46.3  11.7  28.3   7.7    0    0   
21        거주지           충청도   157   7.9  42.2  13.3  20.5   7.9    3  3.6   
22        거주지           전라도   136  11.2  32.6   9.4  29.8   5.2  4.5    1   
23        거주지           경상도   192   8.9  36.4  16.8  26.1   4.3  2.7  3.2   
24        거주지  해외거주 등 기타 지역     6  42.6     0  22.3  35.1     0    0    0   
25    이용 교통수단            항공  3364  10.6  41.7  17.6  19.9   5.2  1.7  1.1   
26    이용 교통수단            선박   119    10  30.2  27.2    17  11.7  2.1  1.1   
27         성별            남자  1450  10.9  37.6  18.6  22.1   5.2  1.9  1.2   
28         성별            여자  2033  10.4  43.9  17.5  18.2   5.5  1.6    1   
29         연령        15-20세    56   3.9  30.4   7.6    45  13.1    0    0   
30         연령        21-30세   733  13.2  51.2  13.8  15.1   3.7  1.4  0.7   
31         연령        31-40세  1196  12.1  46.8  18.8  15.7   3.5  1.1  0.8   
32         연령        41-50세   848   7.9  29.7  23.3  25.8   7.9  2.6  0.8   
33         연령        51-60세   498   9.2    34  15.5  25.4     6  2.6  2.7   
34         연령        61세 이상   153   8.4  42.8  13.4  13.9   8.8  1.6  2.2   
35      여행 목적     휴가 및 순수여행  3099   9.2  42.5  18.3    20   5.5  1.7    1   
36      여행 목적       비즈니스 여행   107  33.4  25.4  10.3  15.8     3  1.9  1.3   
37      여행 목적            기타   277  17.2  33.5  16.9  19.8   4.9  2.5  1.8   
38       여행형태    개별여행(자유여행)  3213  10.7    42  17.8  19.7   5.3  1.6  1.1   
39       여행형태         패키지여행    48  23.6  36.8   9.8   3.9     0  2.4  1.6   
40       여행형태       부분패키지여행   222   5.8  31.5  22.1  25.3   8.2    3  0.5   
41       방문횟수            1회   695  12.3  41.9  15.1  17.7   5.9    2  2.3   
42       방문횟수            2회  1136     8  42.5  17.8  21.4   4.7    2    1   
43       방문횟수            3회   682   8.2  41.6  19.6  20.2   5.9  1.7  0.3   
44       방문횟수         4회 이상   969  14.1  39.3  18.9  19.2   5.4  1.2  0.9   
45  BASE : 전체           NaN   NaN   NaN   NaN   NaN   NaN   NaN  NaN  NaN   

     8명   9명  10명  11명 이상    평균  
0    8명   9명  10명  11명 이상    평균  
1   0.8  0.2  0.5     0.7  2.98  
2     1  0.3  0.8     1.4  3.17  
3   0.7  0.2  0.3     0.5  2.91  
4   0.7  0.4  1.2       2  3.26  
5   1.5    0  0.2     0.2     3  
6   0.6  0.4  0.2     0.2  2.92  
7   0.4  0.2  0.2       0  2.67  
8   0.8    0  0.6     0.8  2.94  
9     1    0  0.4       1   3.1  
10  0.6  0.2

In [ ]:
expenditure_standard_2020 =expenditure_standard_2020.drop(index = range(11, 46)).reset_index(drop=True)

expenditure_standard_2020

KeyError: ignored

1) 월별 패턴:

- 1월과 2월에는 11명 이상의 그룹이 비교적 많이 나타나고, 9월과 10월에는 2명에서 5명까지의 그룹이 더 많음.

2) 평균

- 전체 데이터에 대한 평균값을 보면 "01-02월"의 경우 약 3.17이고, "08-12월"의 경우 약 2.91입니다. 이를 통해 "01-02월"에는 더 많은 인원 수가 있는 경향이 있음.

In [ ]:
expenditure_standard_2020  = expenditure_standard_2020.drop(index = [0, 1]).reset_index(drop=True)

expenditure_standard_2020

분석 카테고리       세부 카테고리   사례수    1명    2명    3명    4명    5명   6명   7명  \
0          구분        01-02월   928    11    40  13.2  22.5   5.8  1.9    2   
1          구분        08-12월  2555  10.4  41.7  19.7  18.8   5.2  1.7  0.7   
2          월별           01월   602  12.8  37.5  13.4  22.3   5.8  1.3  2.5   
3          월별           02월   327   7.7  44.6  12.9  22.9   5.8  2.9  1.2   
4          월별           08월   614   8.9    39  19.6  24.8   4.8    1  0.6   
5          월별           09월   396  14.4  44.4    18  13.5   5.5  2.7  0.8   
6          월별           10월   584    12  40.7  20.9  15.9   5.1  2.4  0.8   
7          월별           11월   620   7.9  43.4  20.7  16.8   6.2  1.8    1   
8          월별           12월   341  10.6  42.3  17.9    23   4.1  0.6  0.4   
9         거주지         서울특별시  1259  12.9  41.1  19.3  18.4   5.8  1.2  0.3   
10        거주지         부산광역시   262  10.8  42.9  14.8  20.8   3.9  2.1  0.4   
11        거주지         대구광역시   139   5.9  49.8  18.3  15.6   2.8  0.6  0.9   
12        거주지         인천광역시   215   9.3  42.4  22.6  15.2   3.9  0.6  2.9   
13        거주지         광주광역시    95  20.5  35.3  11.3  13.5   6.4  1.4  4.9   
14        거주지         대전광역시    76  15.5  41.9  16.8  13.8   8.1  2.4  1.6   
15        거주지         울산광역시    52   1.5  58.9   6.6  24.4   8.5    0    0   
16        거주지         세종자치시    28   4.8  23.5  32.4  13.9  11.9  4.1    0   
17        거주지           경기도   832   8.2    42  19.4  21.1   4.9  2.2  0.7   
18        거주지           강원도    33     6  46.3  11.7  28.3   7.7    0    0   
19        거주지           충청도   157   7.9  42.2  13.3  20.5   7.9    3  3.6   
20        거주지           전라도   136  11.2  32.6   9.4  29.8   5.2  4.5    1   
21        거주지           경상도   192   8.9  36.4  16.8  26.1   4.3  2.7  3.2   
22        거주지  해외거주 등 기타 지역     6  42.6     0  22.3  35.1     0    0    0   
23    이용 교통수단            항공  3364  10.6  41.7  17.6  19.9   5.2  1.7  1.1   
24    이용 교통수단            선박   119    10  30.2  27.2    17  11.7  2.1  1.1   
25         성별            남자  1450  10.9  37.6  18.6  22.1   5.2  1.9  1.2   
26         성별            여자  2033  10.4  43.9  17.5  18.2   5.5  1.6    1   
27         연령        15-20세    56   3.9  30.4   7.6    45  13.1    0    0   
28         연령        21-30세   733  13.2  51.2  13.8  15.1   3.7  1.4  0.7   
29         연령        31-40세  1196  12.1  46.8  18.8  15.7   3.5  1.1  0.8   
30         연령        41-50세   848   7.9  29.7  23.3  25.8   7.9  2.6  0.8   
31         연령        51-60세   498   9.2    34  15.5  25.4     6  2.6  2.7   
32         연령        61세 이상   153   8.4  42.8  13.4  13.9   8.8  1.6  2.2   
33      여행 목적     휴가 및 순수여행  3099   9.2  42.5  18.3    20   5.5  1.7    1   
34      여행 목적       비즈니스 여행   107  33.4  25.4  10.3  15.8     3  1.9  1.3   
35      여행 목적            기타   277  17.2  33.5  16.9  19.8   4.9  2.5  1.8   
36       여행형태    개별여행(자유여행)  3213  10.7    42  17.8  19.7   5.3  1.6  1.1   
37       여행형태         패키지여행    48  23.6  36.8   9.8   3.9     0  2.4  1.6   
38       여행형태       부분패키지여행   222   5.8  31.5  22.1  25.3   8.2    3  0.5   
39       방문횟수            1회   695  12.3  41.9  15.1  17.7   5.9    2  2.3   
40       방문횟수            2회  1136     8  42.5  17.8  21.4   4.7    2    1   
41       방문횟수            3회   682   8.2  41.6  19.6  20.2   5.9  1.7  0.3   
42       방문횟수         4회 이상   969  14.1  39.3  18.9  19.2   5.4  1.2  0.9   
43  BASE : 전체           NaN   NaN   NaN   NaN   NaN   NaN   NaN  NaN  NaN   

     8명   9명  10명 11명 이상    평균  
0     1  0.3  0.8    1.4  3.17  
1   0.7  0.2  0.3    0.5  2.91  
2   0.7  0.4  1.2      2  3.26  
3   1.5    0  0.2    0.2     3  
4   0.6  0.4  0.2    0.2  2.92  
5   0.4  0.2  0.2      0  2.67  
6   0.8    0  0.6    0.8  2.94  
7     1    0  0.4      1   3.1  
8   0.6  0.2  0.2      0  2.77  
9   0.6    0  0.2    0.3  2.77  
10  0.9  0.3  2.1    0.9  3.03  
11  3.5  1.7    0      1  3.03  
12  1.4  0.4    0    1.2     3  
13    0    0  2.7      4  3.42  
14    0    0    0      0  2.71  
15    0    0

## 1인당 지출 경비 - 전체 비용
- 해당 파일에 대한 전처리

In [ ]:
total_cost_2020 = pd.read_excel('/content/drive/MyDrive/2020 1인당 지출 경비 전체.xlsx')
total_cost_2020

table    Unnamed: 1 Unnamed: 2 Unnamed: 3    Unnamed: 4  \
0         NaN           NaN        사례수    10만원 미만  10만원-30만원 미만   
1        [전체]           NaN       3483        3.5          22.6   
2          구분        01-02월        928        0.7          24.7   
3          구분        08-12월       2555        4.6          21.8   
4          월별           01월        602        1.1          24.8   
5          월별           02월        327          0          24.5   
6          월별           08월        614        2.9          17.1   
7          월별           09월        396        1.8          23.5   
8          월별           10월        584        6.9          22.6   
9          월별           11월        620          6          23.5   
10         월별           12월        341        4.1          23.8   
11        거주지         서울특별시       1259        3.6          20.9   
12        거주지         부산광역시        262        6.3          26.3   
13        거주지         대구광역시        139          0          29.2   
14        거주지         인천광역시        215        1.9            23   
15        거주지         광주광역시         95        1.4          30.2   
16        거주지         대전광역시         76        7.7          23.9   
17        거주지         울산광역시         52          0          12.8   
18        거주지         세종자치시         28        4.3           3.8   
19        거주지           경기도        832        3.4            22   
20        거주지           강원도         33          0            31   
21        거주지           충청도        157        1.2          23.2   
22        거주지           전라도        136        5.7          30.5   
23        거주지           경상도        192        5.4          18.3   
24        거주지  해외거주 등 기타 지역          6       11.1          35.1   
25    이용 교통수단            항공       3364        3.4          22.7   
26    이용 교통수단            선박        119        7.4          19.6   
27         성별            남자       1450        3.7          22.2   
28         성별            여자       2033        3.4          22.8   
29         연령        15-20세         56        3.4          32.8   
30         연령        21-30세        733        3.9          25.3   
31         연령        31-40세       1196          3            20   
32         연령        41-50세        848        2.7          23.5   
33         연령        51-60세        498        5.4          21.5   
34         연령        61세 이상        153        4.5          23.9   
35      여행 목적     휴가 및 순수여행       3099        3.1          21.7   
36      여행 목적       비즈니스 여행        107        6.6            37   
37      여행 목적            기타        277        7.3          27.2   
38       여행형태    개별여행(자유여행)       3213        3.1          22.2   
39       여행형태         패키지여행         48       25.5          25.1   
40       여행형태       부분패키지여행        222        4.9          27.2   
41       방문횟수            1회        695        4.6          24.9   
42       방문횟수            2회       1136        3.4          20.2   
43       방문횟수            3회        682        3.6          22.5   
44       방문횟수         4회 이상        969        2.9          23.7   
45  BASE : 전체           NaN        NaN        NaN           NaN   

         Unnamed: 5        Unnamed: 6         Unnamed: 7 Unnamed: 8 Unnamed: 9  
0   30만원 이상-50만원 미만  50만원 이상-100만원 미만  100만원 이상-150만원 미만   150만원 이상         평균  
1              35.7              31.3                4.4        2.5  506343.82  
2              51.4              21.5                1.3        0.5  416735.49  
3              30.1              34.8                5.5        3.2  538901.64  
4              52.1              20.4                1.4        0.2  404989.22  
5                50              23.6                1.1        0.9  438372.54  
6              26.9              42.1                  7        3.9  604136.76  
7              29.2              36.4                6.5        2.7  561786.63  
8              35.2              28.8                3.9        2.6  488937.36  
9              29.7              32.8           

In [ ]:
columns = {'table' : '분석 카테고리',
           'Unnamed: 1' : '세부 카테고리',
           'Unnamed: 2' : '사례수',
           'Unnamed: 3' : '10만원 미만',
           'Unnamed: 4' : '10만원 이상-30만원 미만',
           'Unnamed: 5' : '30만원 이상-50만원 미만',
           'Unnamed: 6' : '50만원 이상-100만원 미만',
           'Unnamed: 7' : '100만원 이상-150만원 미만',
           'Unnamed: 8' : '150만원 이상',
           'Unnamed: 9' : '평균'}

total_cost_2020 = total_cost_2020.rename(columns = columns)
total_cost_2020

분석 카테고리       세부 카테고리   사례수  10만원 미만 10만원 이상-30만원 미만  30만원 이상-50만원 미만  \
0         NaN           NaN   사례수  10만원 미만    10만원-30만원 미만  30만원 이상-50만원 미만   
1        [전체]           NaN  3483      3.5            22.6             35.7   
2          구분        01-02월   928      0.7            24.7             51.4   
3          구분        08-12월  2555      4.6            21.8             30.1   
4          월별           01월   602      1.1            24.8             52.1   
5          월별           02월   327        0            24.5               50   
6          월별           08월   614      2.9            17.1             26.9   
7          월별           09월   396      1.8            23.5             29.2   
8          월별           10월   584      6.9            22.6             35.2   
9          월별           11월   620        6            23.5             29.7   
10         월별           12월   341      4.1            23.8             28.5   
11        거주지         서울특별시  1259      3.6            20.9             33.7   
12        거주지         부산광역시   262      6.3            26.3             33.2   
13        거주지         대구광역시   139        0            29.2             34.9   
14        거주지         인천광역시   215      1.9              23             42.6   
15        거주지         광주광역시    95      1.4            30.2             41.5   
16        거주지         대전광역시    76      7.7            23.9             30.6   
17        거주지         울산광역시    52        0            12.8             40.3   
18        거주지         세종자치시    28      4.3             3.8             41.8   
19        거주지           경기도   832      3.4              22             36.4   
20        거주지           강원도    33        0              31             26.2   
21        거주지           충청도   157      1.2            23.2             40.7   
22        거주지           전라도   136      5.7            30.5             37.8   
23        거주지           경상도   192      5.4            18.3               36   
24        거주지  해외거주 등 기타 지역     6     11.1            35.1             33.4   
25    이용 교통수단            항공  3364      3.4            22.7               36   
26    이용 교통수단            선박   119      7.4            19.6             26.8   
27         성별            남자  1450      3.7            22.2             36.1   
28         성별            여자  2033      3.4            22.8             35.4   
29         연령        15-20세    56      3.4            32.8             55.2   
30         연령        21-30세   733      3.9            25.3               40   
31         연령        31-40세  1196        3              20             32.2   
32         연령        41-50세   848      2.7            23.5             34.7   
33         연령        51-60세   498      5.4            21.5             37.5   
34         연령        61세 이상   153      4.5            23.9             35.6   
35      여행 목적     휴가 및 순수여행  3099      3.1            21.7             36.6   
36      여행 목적       비즈니스 여행   107      6.6              37               35   
37      여행 목적            기타   277      7.3            27.2             26.9   
38       여행형태    개별여행(자유여행)  3213      3.1            22.2             36.1   
39       여행형태         패키지여행    48     25.5            25.1             30.7   
40       여행형태       부분패키지여행   222      4.9            27.2             32.2   
41       방문횟수            1회   695      4.6            24.9             35.7   
42       방문횟수            2회  1136      3.4            20.2             38.9   
43       방문횟수            3회   682      3.6            22.5             37.7   
44       방문횟수         4회 이상   969      2.9            23.7             30.6   
45  BASE : 전체           NaN   NaN      NaN             NaN              NaN   

    50만원 이상-100만원 미만  100만원 이상-150만원 미만  150만원 이상         평균  
0   50만원 이상-100만원 미만  100만원 이상-150만원 미만  150만원 이상         평균  
1               31.3                4.4       2.5  506343.82  
2               21.5                1.3       0.5  416735.49  
3               34.8                5.5 

In [ ]:

total_cost_2020 = total_cost_2020.drop(index = range(11, 46)).reset_index(drop=True)

total_cost_2020

분석 카테고리 세부 카테고리   사례수  10만원 미만 10만원 이상-30만원 미만  30만원 이상-50만원 미만  \
0      NaN     NaN   사례수  10만원 미만    10만원-30만원 미만  30만원 이상-50만원 미만   
1     [전체]     NaN  3483      3.5            22.6             35.7   
2       구분  01-02월   928      0.7            24.7             51.4   
3       구분  08-12월  2555      4.6            21.8             30.1   
4       월별     01월   602      1.1            24.8             52.1   
5       월별     02월   327        0            24.5               50   
6       월별     08월   614      2.9            17.1             26.9   
7       월별     09월   396      1.8            23.5             29.2   
8       월별     10월   584      6.9            22.6             35.2   
9       월별     11월   620        6            23.5             29.7   
10      월별     12월   341      4.1            23.8             28.5   

    50만원 이상-100만원 미만  100만원 이상-150만원 미만  150만원 이상         평균  
0   50만원 이상-100만원 미만  100만원 이상-150만원 미만  150만원 이상         평균  
1               31.3                4.4       2.5  506343.82  
2               21.5                1.3       0.5  416735.49  
3               34.8                5.5       3.2  538901.64  
4               20.4                1.4       0.2  404989.22  
5               23.6                1.1       0.9  438372.54  
6               42.1                  7       3.9  604136.76  
7               36.4                6.5       2.7  561786.63  
8               28.8                3.9       2.6  488937.36  
9               32.8                4.9       3.2  504139.82  
10              34.2                5.3       3.9  543713.69

1) 월별 패턴:

- 1월과 2월은 50만원 이상-100만원 미만의 범주가 높고, 8월은 50만원 이상-100만원 미만의 범주가 가장 높음.

2) 평균값:

- 전체 데이터에 대한 평균값을 보면 "08-12월"에는 더 높은 평균 소비액이 있음.





In [ ]:
total_cost_2020 = total_cost_2020.drop(index = [0, 1]).reset_index(drop=True)

total_cost_2020

분석 카테고리 세부 카테고리   사례수 10만원 미만 10만원 이상-30만원 미만 30만원 이상-50만원 미만  \
0      구분  01-02월   928     0.7            24.7            51.4   
1      구분  08-12월  2555     4.6            21.8            30.1   
2      월별     01월   602     1.1            24.8            52.1   
3      월별     02월   327       0            24.5              50   
4      월별     08월   614     2.9            17.1            26.9   
5      월별     09월   396     1.8            23.5            29.2   
6      월별     10월   584     6.9            22.6            35.2   
7      월별     11월   620       6            23.5            29.7   
8      월별     12월   341     4.1            23.8            28.5   

  50만원 이상-100만원 미만 100만원 이상-150만원 미만 150만원 이상         평균  
0             21.5               1.3      0.5  416735.49  
1             34.8               5.5      3.2  538901.64  
2             20.4               1.4      0.2  404989.22  
3             23.6               1.1      0.9  438372.54  
4             42.1                 7      3.9  604136.76  
5             36.4               6.5      2.7  561786.63  
6             28.8               3.9      2.6  488937.36  
7             32.8               4.9      3.2  504139.82  
8             34.2               5.3      3.9  543713.69

## 1인당 지출 경비 - 숙박 비용
- 해당 파일에 대한 전처리

In [ ]:
accommodation_cost_2020 = pd.read_excel('/content/drive/MyDrive/2020 1인당 지출 경비 - 숙박비.xlsx')
accommodation_cost_2020

table    Unnamed: 1 Unnamed: 2 Unnamed: 3     Unnamed: 4  \
0            NaN           NaN        사례수     5만원 미만  5만원 이상-7만원 미만   
1           [전체]           NaN       3213       20.4           17.5   
2             구분        01-02월        912       28.1           31.3   
3             구분        08-12월       2301       17.3             12   
4             월별           01월        586       28.9           30.7   
5             월별           02월        326       26.6           32.2   
6             월별           08월        543       18.5            8.4   
7             월별           09월        358       12.1            8.9   
8             월별           10월        537       19.1           15.8   
9             월별           11월        558       18.4           12.7   
10            월별           12월        304       16.1           13.9   
11           거주지         서울특별시       1133       18.7           14.4   
12           거주지         부산광역시        251       22.9           20.8   
13           거주지         대구광역시        134       24.3             16   
14           거주지         인천광역시        201       21.4           27.9   
15           거주지         광주광역시         91       18.2           24.8   
16           거주지         대전광역시         70         28           19.4   
17           거주지         울산광역시         43       18.6           18.2   
18           거주지         세종자치시         27        7.5           18.3   
19           거주지           경기도        776       17.8           17.4   
20           거주지           강원도         32       13.9           17.2   
21           거주지           충청도        150       28.6           13.2   
22           거주지           전라도        126       28.4           24.6   
23           거주지           경상도        174         24           15.6   
24           거주지  해외거주 등 기타 지역          5          0           25.1   
25       이용 교통수단            항공       3105       20.6           17.6   
26       이용 교통수단            선박        108       13.6           13.3   
27            성별            남자       1311       21.3           17.8   
28            성별            여자       1902       19.7           17.2   
29            연령        15-20세         56         25           26.1   
30            연령        21-30세        704       22.9           21.5   
31            연령        31-40세       1129       15.5           14.9   
32            연령        41-50세        755       18.5           15.6   
33            연령        51-60세        438       28.6           17.2   
34            연령        61세 이상        130       29.8           25.4   
35         여행 목적     휴가 및 순수여행       2882       20.4           17.8   
36         여행 목적       비즈니스 여행         90         32           19.8   
37         여행 목적            기타        241         16           12.4   
38          여행형태    개별여행(자유여행)       3213       20.4           17.5   
39          방문횟수            1회        645       22.2           19.6   
40          방문횟수            2회       1070       21.6           22.2   
41          방문횟수            3회        621       20.9           16.1   
42          방문횟수         4회 이상        878       17.1           11.1   
43  BASE : 문13=1           NaN        NaN        NaN            NaN   

        Unnamed: 5       Unnamed: 6       Unnamed: 7 Unnamed: 8 Unnamed: 9  \
0   7만원 이상-10만원 미만  10만원 이상-20만원 미만  20만원 이상-30만원 미만    30만원 이상       경험없음   
1               12             26.6              8.3        7.1        8.2   
2             14.2             15.5              1.9        1.8        7.2   
3             11.1               31             10.8        9.2        8.6   
4             13.1             13.7              2.4        1.8        9.4   
5             16.1             18.9              1.1        1.8        3.3   
6              9.8             32.4             11.8         11          8   
7              9.5             34.4             14.9        9.7       10.5   
8             12.6             26.5             11.1          6        8.8   
9             12.7             31.4

In [ ]:
columns = {'table' : '분석 카테고리',
           'Unnamed: 1' : '세부 카테고리',
           'Unnamed: 2' : '사례수',
           'Unnamed: 3' : '5만원 미만',
           'Unnamed: 4' : '5만원 이상-7만원 미만',
           'Unnamed: 5' : '7만원 이상-10만원 미만',
           'Unnamed: 6' : '10만원 이상-20만원 미만',
           'Unnamed: 7' : '20만원 이상-30만원 미만',
           'Unnamed: 8' : '30만원 이상',
           'Unnamed: 9' : '경험없음',
           'Unnamed: 10' : '평균'}

accommodation_cost_2020 = accommodation_cost_2020.rename(columns = columns)
accommodation_cost_2020

분석 카테고리       세부 카테고리   사례수  5만원 미만  5만원 이상-7만원 미만  7만원 이상-10만원 미만  \
0            NaN           NaN   사례수  5만원 미만  5만원 이상-7만원 미만  7만원 이상-10만원 미만   
1           [전체]           NaN  3213    20.4           17.5              12   
2             구분        01-02월   912    28.1           31.3            14.2   
3             구분        08-12월  2301    17.3             12            11.1   
4             월별           01월   586    28.9           30.7            13.1   
5             월별           02월   326    26.6           32.2            16.1   
6             월별           08월   543    18.5            8.4             9.8   
7             월별           09월   358    12.1            8.9             9.5   
8             월별           10월   537    19.1           15.8            12.6   
9             월별           11월   558    18.4           12.7            12.7   
10            월별           12월   304    16.1           13.9             9.9   
11           거주지         서울특별시  1133    18.7           14.4            10.7   
12           거주지         부산광역시   251    22.9           20.8            11.1   
13           거주지         대구광역시   134    24.3             16             6.8   
14           거주지         인천광역시   201    21.4           27.9            13.6   
15           거주지         광주광역시    91    18.2           24.8              17   
16           거주지         대전광역시    70      28           19.4            13.5   
17           거주지         울산광역시    43    18.6           18.2              20   
18           거주지         세종자치시    27     7.5           18.3            19.6   
19           거주지           경기도   776    17.8           17.4            12.9   
20           거주지           강원도    32    13.9           17.2            20.4   
21           거주지           충청도   150    28.6           13.2            15.2   
22           거주지           전라도   126    28.4           24.6            13.2   
23           거주지           경상도   174      24           15.6             8.6   
24           거주지  해외거주 등 기타 지역     5       0           25.1               0   
25       이용 교통수단            항공  3105    20.6           17.6              12   
26       이용 교통수단            선박   108    13.6           13.3            11.3   
27            성별            남자  1311    21.3           17.8            12.8   
28            성별            여자  1902    19.7           17.2            11.4   
29            연령        15-20세    56      25           26.1            21.2   
30            연령        21-30세   704    22.9           21.5            12.7   
31            연령        31-40세  1129    15.5           14.9            11.2   
32            연령        41-50세   755    18.5           15.6            11.1   
33            연령        51-60세   438    28.6           17.2            13.9   
34            연령        61세 이상   130    29.8           25.4             9.8   
35         여행 목적     휴가 및 순수여행  2882    20.4           17.8            12.2   
36         여행 목적       비즈니스 여행    90      32           19.8             5.4   
37         여행 목적            기타   241      16           12.4            11.9   
38          여행형태    개별여행(자유여행)  3213    20.4           17.5              12   
39          방문횟수            1회   645    22.2           19.6            14.3   
40          방문횟수            2회  1070    21.6           22.2            12.9   
41          방문횟수            3회   621    20.9           16.1            11.2   
42          방문횟수         4회 이상   878    17.1           11.1             9.8   
43  BASE : 문13=1           NaN   NaN     NaN            NaN             NaN   

    10만원 이상-20만원 미만  20만원 이상-30만원 미만  30만원 이상  경험없음         평균  
0   10만원 이상-20만원 미만  20만원 이상-30만원 미만  30만원 이상  경험없음         평균  
1              26.6              8.3      7.1   8.2  115035.87  
2              15.5              1.9      1.8   7.2   69572.37  
3                31             10.8      9.2   8.6  133047.46  
4              13.7              2.4      1.8   9.4   66994.59  
5              18.9              1.1      1.8   3.3   74199.66 

In [ ]:
accommodation_cost_2020 = accommodation_cost_2020.drop(index = [0, 1]).reset_index(drop=True)

accommodation_cost_2020

분석 카테고리       세부 카테고리   사례수 5만원 미만 5만원 이상-7만원 미만 7만원 이상-10만원 미만  \
0             구분        01-02월   912   28.1          31.3           14.2   
1             구분        08-12월  2301   17.3            12           11.1   
2             월별           01월   586   28.9          30.7           13.1   
3             월별           02월   326   26.6          32.2           16.1   
4             월별           08월   543   18.5           8.4            9.8   
5             월별           09월   358   12.1           8.9            9.5   
6             월별           10월   537   19.1          15.8           12.6   
7             월별           11월   558   18.4          12.7           12.7   
8             월별           12월   304   16.1          13.9            9.9   
9            거주지         서울특별시  1133   18.7          14.4           10.7   
10           거주지         부산광역시   251   22.9          20.8           11.1   
11           거주지         대구광역시   134   24.3            16            6.8   
12           거주지         인천광역시   201   21.4          27.9           13.6   
13           거주지         광주광역시    91   18.2          24.8             17   
14           거주지         대전광역시    70     28          19.4           13.5   
15           거주지         울산광역시    43   18.6          18.2             20   
16           거주지         세종자치시    27    7.5          18.3           19.6   
17           거주지           경기도   776   17.8          17.4           12.9   
18           거주지           강원도    32   13.9          17.2           20.4   
19           거주지           충청도   150   28.6          13.2           15.2   
20           거주지           전라도   126   28.4          24.6           13.2   
21           거주지           경상도   174     24          15.6            8.6   
22           거주지  해외거주 등 기타 지역     5      0          25.1              0   
23       이용 교통수단            항공  3105   20.6          17.6             12   
24       이용 교통수단            선박   108   13.6          13.3           11.3   
25            성별            남자  1311   21.3          17.8           12.8   
26            성별            여자  1902   19.7          17.2           11.4   
27            연령        15-20세    56     25          26.1           21.2   
28            연령        21-30세   704   22.9          21.5           12.7   
29            연령        31-40세  1129   15.5          14.9           11.2   
30            연령        41-50세   755   18.5          15.6           11.1   
31            연령        51-60세   438   28.6          17.2           13.9   
32            연령        61세 이상   130   29.8          25.4            9.8   
33         여행 목적     휴가 및 순수여행  2882   20.4          17.8           12.2   
34         여행 목적       비즈니스 여행    90     32          19.8            5.4   
35         여행 목적            기타   241     16          12.4           11.9   
36          여행형태    개별여행(자유여행)  3213   20.4          17.5             12   
37          방문횟수            1회   645   22.2          19.6           14.3   
38          방문횟수            2회  1070   21.6          22.2           12.9   
39          방문횟수            3회   621   20.9          16.1           11.2   
40          방문횟수         4회 이상   878   17.1          11.1            9.8   
41  BASE : 문13=1           NaN   NaN    NaN           NaN            NaN   

   10만원 이상-20만원 미만 20만원 이상-30만원 미만 30만원 이상  경험없음         평균  
0             15.5             1.9     1.8   7.2   69572.37  
1               31            10.8     9.2   8.6  133047.46  
2             13.7             2.4     1.8   9.4   66994.59  
3             18.9             1.1     1.8   3.3   74199.66  
4             32.4            11.8      11     8  141989.82  
5             34.4            14.9     9.7  10.5  156554.73  
6             26.5            11.1       6   8.8  113934.89  
7             31.4             7.2       9   8.5  123836.35  
8             31.3            10.4    11.3   7.1  140040.31  
9             28.3            10.4     8.9   8.6  128014.42  
10            22.2             7.8       6   9.2   98430.1

In [ ]:
accommodation_cost_2020 = accommodation_cost_2020.drop(index = range(9, 42)).reset_index(drop=True)

accommodation_cost_2020

분석 카테고리 세부 카테고리   사례수 5만원 미만 5만원 이상-7만원 미만 7만원 이상-10만원 미만 10만원 이상-20만원 미만  \
0      구분  01-02월   912   28.1          31.3           14.2            15.5   
1      구분  08-12월  2301   17.3            12           11.1              31   
2      월별     01월   586   28.9          30.7           13.1            13.7   
3      월별     02월   326   26.6          32.2           16.1            18.9   
4      월별     08월   543   18.5           8.4            9.8            32.4   
5      월별     09월   358   12.1           8.9            9.5            34.4   
6      월별     10월   537   19.1          15.8           12.6            26.5   
7      월별     11월   558   18.4          12.7           12.7            31.4   
8      월별     12월   304   16.1          13.9            9.9            31.3   

  20만원 이상-30만원 미만 30만원 이상  경험없음         평균  
0             1.9     1.8   7.2   69572.37  
1            10.8     9.2   8.6  133047.46  
2             2.4     1.8   9.4   66994.59  
3             1.1     1.8   3.3   74199.66  
4            11.8      11     8  141989.82  
5            14.9     9.7  10.5  156554.73  
6            11.1       6   8.8  113934.89  
7             7.2       9   8.5  123836.35  
8            10.4    11.3   7.1  140040.31

## 1인당 지출 경비 - 식음료
- 해당 파일에 대한 전처리

In [ ]:
food_cost_2020 = pd.read_excel('/content/drive/MyDrive/2020 1인당 지출 경비 - 식음료.xlsx')
food_cost_2020

table    Unnamed: 1 Unnamed: 2 Unnamed: 3    Unnamed: 4  \
0            NaN           NaN        사례수     3만원 미만  3만원이상-5만원 미만   
1           [전체]           NaN       3213        7.4           4.3   
2             구분        01-02월        912          3           2.1   
3             구분        08-12월       2301        9.2           5.2   
4             월별           01월        586        3.9           2.2   
5             월별           02월        326        1.3             2   
6             월별           08월        543        7.8           5.6   
7             월별           09월        358        6.6           3.7   
8             월별           10월        537       10.1             6   
9             월별           11월        558       10.3           5.3   
10            월별           12월        304         11           4.9   
11           거주지         서울특별시       1133        7.2           4.6   
12           거주지         부산광역시        251        8.8           5.1   
13           거주지         대구광역시        134        4.1           3.8   
14           거주지         인천광역시        201        7.2           3.9   
15           거주지         광주광역시         91        6.4           7.2   
16           거주지         대전광역시         70       14.1           9.7   
17           거주지         울산광역시         43        2.7           2.8   
18           거주지         세종자치시         27        4.3           4.5   
19           거주지           경기도        776        7.1           3.4   
20           거주지           강원도         32        3.8           7.6   
21           거주지           충청도        150        8.9             1   
22           거주지           전라도        126        8.8           3.9   
23           거주지           경상도        174        8.4           5.3   
24           거주지  해외거주 등 기타 지역          5       14.5          12.5   
25       이용 교통수단            항공       3105        7.4           4.4   
26       이용 교통수단            선박        108          8           0.5   
27            성별            남자       1311          7           4.9   
28            성별            여자       1902        7.7           3.9   
29            연령        15-20세         56        5.8           3.4   
30            연령        21-30세        704        7.3           4.3   
31            연령        31-40세       1129        7.5           4.4   
32            연령        41-50세        755        6.4           4.6   
33            연령        51-60세        438       10.5           4.3   
34            연령        61세 이상        130          3           2.8   
35         여행 목적     휴가 및 순수여행       2882        6.6           3.7   
36         여행 목적       비즈니스 여행         90       13.2          12.6   
37         여행 목적            기타        241       14.6           8.5   
38          여행형태    개별여행(자유여행)       3213        7.4           4.3   
39          방문횟수            1회        645        8.9           4.1   
40          방문횟수            2회       1070        6.1           3.1   
41          방문횟수            3회        621        6.9           5.3   
42          방문횟수         4회 이상        878        8.3           5.3   
43  BASE : 문13=1           NaN        NaN        NaN           NaN   

        Unnamed: 5       Unnamed: 6 Unnamed: 7 Unnamed: 8 Unnamed: 9  
0   5만원 이상-10만원 미만  10만원 이상-20만원 미만    20만원 이상       경험없음         평균  
1             22.2             47.7       17.7        0.7  130560.99  
2             21.2             64.4        8.8        0.5  119074.37  
3             22.6             41.1       21.1        0.8  135111.73  
4             23.9             62.4        7.1        0.5  112607.71  
5             16.5             67.9         12        0.3  130682.45  
6             19.3             41.7       25.2        0.4  155146.11  
7             21.9             43.8       23.1        0.9  136901.54  
8               21             40.7       21.2        1.1  133537.73  
9               27             39.1       17.5        0.9  122143.76  
10            24.3             41.1       18.1        0.7  123796.42  
11      

In [ ]:

columns = {'table' : '분석 카테고리',
           'Unnamed: 1' : '세부 카테고리',
           'Unnamed: 2' : '사례수',
           'Unnamed: 3' : '3만원 미만',
           'Unnamed: 4' : '3만원이상-5만원 미만',
           'Unnamed: 5' : '5만원 이상-10만원 미만',
           'Unnamed: 6' : '10만원 이상-20만원 미만',
           'Unnamed: 7' : '20만원 이상',
           'Unnamed: 8' : '경험없음',
           'Unnamed: 9' : '평균'}

food_cost_2020 = food_cost_2020.rename(columns = columns)
food_cost_2020

분석 카테고리       세부 카테고리   사례수  3만원 미만  3만원이상-5만원 미만  5만원 이상-10만원 미만  \
0            NaN           NaN   사례수  3만원 미만  3만원이상-5만원 미만  5만원 이상-10만원 미만   
1           [전체]           NaN  3213     7.4           4.3            22.2   
2             구분        01-02월   912       3           2.1            21.2   
3             구분        08-12월  2301     9.2           5.2            22.6   
4             월별           01월   586     3.9           2.2            23.9   
5             월별           02월   326     1.3             2            16.5   
6             월별           08월   543     7.8           5.6            19.3   
7             월별           09월   358     6.6           3.7            21.9   
8             월별           10월   537    10.1             6              21   
9             월별           11월   558    10.3           5.3              27   
10            월별           12월   304      11           4.9            24.3   
11           거주지         서울특별시  1133     7.2           4.6            20.1   
12           거주지         부산광역시   251     8.8           5.1            24.1   
13           거주지         대구광역시   134     4.1           3.8            29.6   
14           거주지         인천광역시   201     7.2           3.9            19.2   
15           거주지         광주광역시    91     6.4           7.2            30.1   
16           거주지         대전광역시    70    14.1           9.7            12.9   
17           거주지         울산광역시    43     2.7           2.8            20.6   
18           거주지         세종자치시    27     4.3           4.5              16   
19           거주지           경기도   776     7.1           3.4            21.3   
20           거주지           강원도    32     3.8           7.6            28.7   
21           거주지           충청도   150     8.9             1            26.6   
22           거주지           전라도   126     8.8           3.9            34.8   
23           거주지           경상도   174     8.4           5.3              23   
24           거주지  해외거주 등 기타 지역     5    14.5          12.5               0   
25       이용 교통수단            항공  3105     7.4           4.4            22.2   
26       이용 교통수단            선박   108       8           0.5            23.6   
27            성별            남자  1311       7           4.9            24.2   
28            성별            여자  1902     7.7           3.9            20.9   
29            연령        15-20세    56     5.8           3.4            30.3   
30            연령        21-30세   704     7.3           4.3            22.2   
31            연령        31-40세  1129     7.5           4.4            18.9   
32            연령        41-50세   755     6.4           4.6            26.3   
33            연령        51-60세   438    10.5           4.3            21.3   
34            연령        61세 이상   130       3           2.8            27.3   
35         여행 목적     휴가 및 순수여행  2882     6.6           3.7            22.2   
36         여행 목적       비즈니스 여행    90    13.2          12.6            34.3   
37         여행 목적            기타   241    14.6           8.5            18.8   
38          여행형태    개별여행(자유여행)  3213     7.4           4.3            22.2   
39          방문횟수            1회   645     8.9           4.1            21.9   
40          방문횟수            2회  1070     6.1           3.1            20.5   
41          방문횟수            3회   621     6.9           5.3            22.1   
42          방문횟수         4회 이상   878     8.3           5.3            24.6   
43  BASE : 문13=1           NaN   NaN     NaN           NaN             NaN   

    10만원 이상-20만원 미만  20만원 이상  경험없음         평균  
0   10만원 이상-20만원 미만  20만원 이상  경험없음         평균  
1              47.7     17.7   0.7  130560.99  
2              64.4      8.8   0.5  119074.37  
3              41.1     21.1   0.8  135111.73  
4              62.4      7.1   0.5  112607.71  
5              67.9       12   0.3  130682.45  
6              41.7     25.2   0.4  155146.11  
7              43.8     23.1   0.9  136901.54  
8              40.7     21.2   1.1  133537.73  
9              39.

In [ ]:
food_cost_2020 = food_cost_2020.drop(index = [0, 1]).reset_index(drop=True)

food_cost_2020

분석 카테고리       세부 카테고리   사례수 3만원 미만 3만원이상-5만원 미만 5만원 이상-10만원 미만  \
0             구분        01-02월   912      3          2.1           21.2   
1             구분        08-12월  2301    9.2          5.2           22.6   
2             월별           01월   586    3.9          2.2           23.9   
3             월별           02월   326    1.3            2           16.5   
4             월별           08월   543    7.8          5.6           19.3   
5             월별           09월   358    6.6          3.7           21.9   
6             월별           10월   537   10.1            6             21   
7             월별           11월   558   10.3          5.3             27   
8             월별           12월   304     11          4.9           24.3   
9            거주지         서울특별시  1133    7.2          4.6           20.1   
10           거주지         부산광역시   251    8.8          5.1           24.1   
11           거주지         대구광역시   134    4.1          3.8           29.6   
12           거주지         인천광역시   201    7.2          3.9           19.2   
13           거주지         광주광역시    91    6.4          7.2           30.1   
14           거주지         대전광역시    70   14.1          9.7           12.9   
15           거주지         울산광역시    43    2.7          2.8           20.6   
16           거주지         세종자치시    27    4.3          4.5             16   
17           거주지           경기도   776    7.1          3.4           21.3   
18           거주지           강원도    32    3.8          7.6           28.7   
19           거주지           충청도   150    8.9            1           26.6   
20           거주지           전라도   126    8.8          3.9           34.8   
21           거주지           경상도   174    8.4          5.3             23   
22           거주지  해외거주 등 기타 지역     5   14.5         12.5              0   
23       이용 교통수단            항공  3105    7.4          4.4           22.2   
24       이용 교통수단            선박   108      8          0.5           23.6   
25            성별            남자  1311      7          4.9           24.2   
26            성별            여자  1902    7.7          3.9           20.9   
27            연령        15-20세    56    5.8          3.4           30.3   
28            연령        21-30세   704    7.3          4.3           22.2   
29            연령        31-40세  1129    7.5          4.4           18.9   
30            연령        41-50세   755    6.4          4.6           26.3   
31            연령        51-60세   438   10.5          4.3           21.3   
32            연령        61세 이상   130      3          2.8           27.3   
33         여행 목적     휴가 및 순수여행  2882    6.6          3.7           22.2   
34         여행 목적       비즈니스 여행    90   13.2         12.6           34.3   
35         여행 목적            기타   241   14.6          8.5           18.8   
36          여행형태    개별여행(자유여행)  3213    7.4          4.3           22.2   
37          방문횟수            1회   645    8.9          4.1           21.9   
38          방문횟수            2회  1070    6.1          3.1           20.5   
39          방문횟수            3회   621    6.9          5.3           22.1   
40          방문횟수         4회 이상   878    8.3          5.3           24.6   
41  BASE : 문13=1           NaN   NaN    NaN          NaN            NaN   

   10만원 이상-20만원 미만 20만원 이상 경험없음         평균  
0             64.4     8.8  0.5  119074.37  
1             41.1    21.1  0.8  135111.73  
2             62.4     7.1  0.5  112607.71  
3             67.9      12  0.3  130682.45  
4             41.7    25.2  0.4  155146.11  
5             43.8    23.1  0.9  136901.54  
6             40.7    21.2  1.1  133537.73  
7             39.1    17.5  0.9  122143.76  
8             41.1    18.1  0.7  123796.42  
9               49    18.2  0.8  134610.51  
10            42.9    17.4  1.6  129682.11  
11            40.1    22.5    0  141633.33  
12              55    14.3  0.4  121235.83  
13            39.6    15.2  1.5  134589.95  
14            38.2      25    0  132294.09  
15            60.8    13.2    0  116851.62  
16            43.4

1) 월별 패턴:

- 1월과 2월은 20만원 이상의 범주가 가장 높고, 8월은 20만원 이상의 범주가 가장 높음.

2) 평균값:

- "08-12월"에는 더 높은 평균 소비액이 있음을 알수 있음.

In [ ]:
food_cost_2020 = food_cost_2020.drop(index = range(9, 42)).reset_index(drop=True)

food_cost_2020

분석 카테고리 세부 카테고리   사례수 3만원 미만 3만원이상-5만원 미만 5만원 이상-10만원 미만 10만원 이상-20만원 미만  \
0      구분  01-02월   912      3          2.1           21.2            64.4   
1      구분  08-12월  2301    9.2          5.2           22.6            41.1   
2      월별     01월   586    3.9          2.2           23.9            62.4   
3      월별     02월   326    1.3            2           16.5            67.9   
4      월별     08월   543    7.8          5.6           19.3            41.7   
5      월별     09월   358    6.6          3.7           21.9            43.8   
6      월별     10월   537   10.1            6             21            40.7   
7      월별     11월   558   10.3          5.3             27            39.1   
8      월별     12월   304     11          4.9           24.3            41.1   

  20만원 이상 경험없음         평균  
0     8.8  0.5  119074.37  
1    21.1  0.8  135111.73  
2     7.1  0.5  112607.71  
3      12  0.3  130682.45  
4    25.2  0.4  155146.11  
5    23.1  0.9  136901.54  
6    21.2  1.1  133537.73  
7    17.5  0.9  122143.76  
8    18.1  0.7  123796.42

## 1인당 지출 경비 - 항공/선박 이용 비용
- 해당 파일에 대한 전처리

In [ ]:
transportation_cost_2020 = pd.read_excel('/content/drive/MyDrive/2020 1인당 지출경비 - 항공료.xlsx')
transportation_cost_2020

table    Unnamed: 1 Unnamed: 2 Unnamed: 3    Unnamed: 4  \
0            NaN           NaN        사례수     3만원 미만  3만원이상-5만원 미만   
1           [전체]           NaN       3213       10.8           8.6   
2             구분        01-02월        912        0.9           3.7   
3             구분        08-12월       2301       14.6          10.6   
4             월별           01월        586        1.2           1.5   
5             월별           02월        326        0.5           7.7   
6             월별           08월        543        9.5           8.3   
7             월별           09월        358       12.8          10.3   
8             월별           10월        537       18.2          11.1   
9             월별           11월        558       14.7          11.2   
10            월별           12월        304       19.7          12.8   
11           거주지         서울특별시       1133       10.7           9.2   
12           거주지         부산광역시        251       13.9          10.4   
13           거주지         대구광역시        134        9.9           9.2   
14           거주지         인천광역시        201        8.7             6   
15           거주지         광주광역시         91        2.7           8.1   
16           거주지         대전광역시         70       18.4          14.2   
17           거주지         울산광역시         43        5.1           7.3   
18           거주지         세종자치시         27          7             8   
19           거주지           경기도        776       12.6           7.7   
20           거주지           강원도         32        7.5           8.9   
21           거주지           충청도        150       10.5           7.8   
22           거주지           전라도        126          7           7.3   
23           거주지           경상도        174          7           9.2   
24           거주지  해외거주 등 기타 지역          5       35.4             0   
25       이용 교통수단            항공       3105         11           8.7   
26       이용 교통수단            선박        108        4.6           6.7   
27            성별            남자       1311        9.6           7.5   
28            성별            여자       1902       11.6           9.4   
29            연령        15-20세         56        5.8           9.1   
30            연령        21-30세        704        9.8           9.2   
31            연령        31-40세       1129       12.4           8.6   
32            연령        41-50세        755        9.9             9   
33            연령        51-60세        438         11           9.1   
34            연령        61세 이상        130        7.2           2.1   
35         여행 목적     휴가 및 순수여행       2882       10.7           8.4   
36         여행 목적       비즈니스 여행         90        9.4           7.5   
37         여행 목적            기타        241       12.3          11.6   
38          여행형태    개별여행(자유여행)       3213       10.8           8.6   
39          방문횟수            1회        645       11.3           8.3   
40          방문횟수            2회       1070          9           7.1   
41          방문횟수            3회        621       12.2             9   
42          방문횟수         4회 이상        878       11.4          10.4   
43  BASE : 문13=1           NaN        NaN        NaN           NaN   

        Unnamed: 5       Unnamed: 6 Unnamed: 7 Unnamed: 8  
0   5만원 이상-10만원 미만  10만원 이상-15만원 미만    15만원 이상         평균  
1             39.2             24.8       16.6   95095.86  
2             61.1             27.8        6.4   88848.07  
3             30.6             23.5       20.7   97571.09  
4             55.1               35        7.3   95715.25  
5             71.9               15        4.8   76521.01  
6             24.9             27.1       30.1  117961.43  
7             33.2             23.3       20.4   98776.17  
8             31.9             21.6       17.2   86454.07  
9             32.7             25.1       16.5   91499.53  
10            31.6             18.1       17.9   90500.34  
11            35.2             27.7       17.1    97798.4  
12            37.1             21.7       16.9   88278.93  
13            43.6  

In [ ]:
columns = {'table' : '분석 카테고리',
           'Unnamed: 1' : '세부 카테고리',
           'Unnamed: 2' : '사례수',
           'Unnamed: 3' : '3만원 미만',
           'Unnamed: 4' : '3만원이상-5만원 미만',
           'Unnamed: 5' : '5만원 이상-10만원 미만',
           'Unnamed: 6' : '10만원 이상-15만원 미만',
           'Unnamed: 7' : '15만원 이상',
           'Unnamed: 8' : '평균'}

transportation_cost_2020 = transportation_cost_2020.rename(columns = columns)
transportation_cost_2020


분석 카테고리       세부 카테고리   사례수  3만원 미만  3만원이상-5만원 미만  5만원 이상-10만원 미만  \
0            NaN           NaN   사례수  3만원 미만  3만원이상-5만원 미만  5만원 이상-10만원 미만   
1           [전체]           NaN  3213    10.8           8.6            39.2   
2             구분        01-02월   912     0.9           3.7            61.1   
3             구분        08-12월  2301    14.6          10.6            30.6   
4             월별           01월   586     1.2           1.5            55.1   
5             월별           02월   326     0.5           7.7            71.9   
6             월별           08월   543     9.5           8.3            24.9   
7             월별           09월   358    12.8          10.3            33.2   
8             월별           10월   537    18.2          11.1            31.9   
9             월별           11월   558    14.7          11.2            32.7   
10            월별           12월   304    19.7          12.8            31.6   
11           거주지         서울특별시  1133    10.7           9.2            35.2   
12           거주지         부산광역시   251    13.9          10.4            37.1   
13           거주지         대구광역시   134     9.9           9.2            43.6   
14           거주지         인천광역시   201     8.7             6              52   
15           거주지         광주광역시    91     2.7           8.1            47.3   
16           거주지         대전광역시    70    18.4          14.2            40.3   
17           거주지         울산광역시    43     5.1           7.3            55.4   
18           거주지         세종자치시    27       7             8            27.1   
19           거주지           경기도   776    12.6           7.7            39.5   
20           거주지           강원도    32     7.5           8.9              43   
21           거주지           충청도   150    10.5           7.8            40.2   
22           거주지           전라도   126       7           7.3            44.6   
23           거주지           경상도   174       7           9.2            36.6   
24           거주지  해외거주 등 기타 지역     5    35.4             0            50.1   
25       이용 교통수단            항공  3105      11           8.7            39.6   
26       이용 교통수단            선박   108     4.6           6.7            29.4   
27            성별            남자  1311     9.6           7.5            41.2   
28            성별            여자  1902    11.6           9.4            37.9   
29            연령        15-20세    56     5.8           9.1            46.7   
30            연령        21-30세   704     9.8           9.2            45.5   
31            연령        31-40세  1129    12.4           8.6            35.1   
32            연령        41-50세   755     9.9             9            39.2   
33            연령        51-60세   438      11           9.1            36.6   
34            연령        61세 이상   130     7.2           2.1              47   
35         여행 목적     휴가 및 순수여행  2882    10.7           8.4            39.6   
36         여행 목적       비즈니스 여행    90     9.4           7.5            40.5   
37         여행 목적            기타   241    12.3          11.6            34.2   
38          여행형태    개별여행(자유여행)  3213    10.8           8.6            39.2   
39          방문횟수            1회   645    11.3           8.3            38.2   
40          방문횟수            2회  1070       9           7.1            43.1   
41          방문횟수            3회   621    12.2             9            39.3   
42          방문횟수         4회 이상   878    11.4          10.4            35.3   
43  BASE : 문13=1           NaN   NaN     NaN           NaN             NaN   

    10만원 이상-15만원 미만  15만원 이상         평균  
0   10만원 이상-15만원 미만  15만원 이상         평균  
1              24.8     16.6   95095.86  
2              27.8      6.4   88848.07  
3              23.5     20.7   97571.09  
4                35      7.3   95715.25  
5                15      4.8   76521.01  
6              27.1     30.1  117961.43  
7              23.3     20.4   98776.17  
8              21.6     17.2   86454.07  
9              25.1     16.5   91499.53  
10             18.1     17.9   90500

In [ ]:
transportation_cost_2020 = transportation_cost_2020.drop(index = [0, 1]).reset_index(drop=True)

transportation_cost_2020

분석 카테고리       세부 카테고리   사례수 3만원 미만 3만원이상-5만원 미만 5만원 이상-10만원 미만  \
0             구분        01-02월   912    0.9          3.7           61.1   
1             구분        08-12월  2301   14.6         10.6           30.6   
2             월별           01월   586    1.2          1.5           55.1   
3             월별           02월   326    0.5          7.7           71.9   
4             월별           08월   543    9.5          8.3           24.9   
5             월별           09월   358   12.8         10.3           33.2   
6             월별           10월   537   18.2         11.1           31.9   
7             월별           11월   558   14.7         11.2           32.7   
8             월별           12월   304   19.7         12.8           31.6   
9            거주지         서울특별시  1133   10.7          9.2           35.2   
10           거주지         부산광역시   251   13.9         10.4           37.1   
11           거주지         대구광역시   134    9.9          9.2           43.6   
12           거주지         인천광역시   201    8.7            6             52   
13           거주지         광주광역시    91    2.7          8.1           47.3   
14           거주지         대전광역시    70   18.4         14.2           40.3   
15           거주지         울산광역시    43    5.1          7.3           55.4   
16           거주지         세종자치시    27      7            8           27.1   
17           거주지           경기도   776   12.6          7.7           39.5   
18           거주지           강원도    32    7.5          8.9             43   
19           거주지           충청도   150   10.5          7.8           40.2   
20           거주지           전라도   126      7          7.3           44.6   
21           거주지           경상도   174      7          9.2           36.6   
22           거주지  해외거주 등 기타 지역     5   35.4            0           50.1   
23       이용 교통수단            항공  3105     11          8.7           39.6   
24       이용 교통수단            선박   108    4.6          6.7           29.4   
25            성별            남자  1311    9.6          7.5           41.2   
26            성별            여자  1902   11.6          9.4           37.9   
27            연령        15-20세    56    5.8          9.1           46.7   
28            연령        21-30세   704    9.8          9.2           45.5   
29            연령        31-40세  1129   12.4          8.6           35.1   
30            연령        41-50세   755    9.9            9           39.2   
31            연령        51-60세   438     11          9.1           36.6   
32            연령        61세 이상   130    7.2          2.1             47   
33         여행 목적     휴가 및 순수여행  2882   10.7          8.4           39.6   
34         여행 목적       비즈니스 여행    90    9.4          7.5           40.5   
35         여행 목적            기타   241   12.3         11.6           34.2   
36          여행형태    개별여행(자유여행)  3213   10.8          8.6           39.2   
37          방문횟수            1회   645   11.3          8.3           38.2   
38          방문횟수            2회  1070      9          7.1           43.1   
39          방문횟수            3회   621   12.2            9           39.3   
40          방문횟수         4회 이상   878   11.4         10.4           35.3   
41  BASE : 문13=1           NaN   NaN    NaN          NaN            NaN   

   10만원 이상-15만원 미만 15만원 이상         평균  
0             27.8     6.4   88848.07  
1             23.5    20.7   97571.09  
2               35     7.3   95715.25  
3               15     4.8   76521.01  
4             27.1    30.1  117961.43  
5             23.3    20.4   98776.17  
6             21.6    17.2   86454.07  
7             25.1    16.5   91499.53  
8             18.1    17.9   90500.34  
9             27.7    17.1    97798.4  
10            21.7    16.9   88278.93  
11            23.3    13.9      89176  
12            19.6    13.7   87892.26  
13            24.8    17.1  100877.15  
14            10.7    16.4    77946.9  
15            12.4    19.8  114719.75  
16            39.2    18.8  103949.37  
17            22.8    17.4   93515.18  
18            25.3    1

In [ ]:
transportation_cost_2020 = transportation_cost_2020.drop(index = range(9, 42)).reset_index(drop=True)

transportation_cost_2020

분석 카테고리 세부 카테고리   사례수 3만원 미만 3만원이상-5만원 미만 5만원 이상-10만원 미만 10만원 이상-15만원 미만  \
0      구분  01-02월   912    0.9          3.7           61.1            27.8   
1      구분  08-12월  2301   14.6         10.6           30.6            23.5   
2      월별     01월   586    1.2          1.5           55.1              35   
3      월별     02월   326    0.5          7.7           71.9              15   
4      월별     08월   543    9.5          8.3           24.9            27.1   
5      월별     09월   358   12.8         10.3           33.2            23.3   
6      월별     10월   537   18.2         11.1           31.9            21.6   
7      월별     11월   558   14.7         11.2           32.7            25.1   
8      월별     12월   304   19.7         12.8           31.6            18.1   

  15만원 이상         평균  
0     6.4   88848.07  
1    20.7   97571.09  
2     7.3   95715.25  
3     4.8   76521.01  
4    30.1  117961.43  
5    20.4   98776.17  
6    17.2   86454.07  
7    16.5   91499.53  
8    17.9   90500.34

## 1인당 지출 경비 - 관광, 문화 지출 비용
- 해당 파일에 대한 전처리

In [ ]:
culture_cost_2020 = pd.read_excel('/content/drive/MyDrive/2020 1인당 지출 경비 - 관광.xlsx')
culture_cost_2020

table    Unnamed: 1 Unnamed: 2 Unnamed: 3     Unnamed: 4  \
0            NaN           NaN        사례수     2만원 미만  2만원 이상-5만원 미만   
1           [전체]           NaN       3213       37.1           20.7   
2             구분        01-02월        912       39.2           16.5   
3             구분        08-12월       2301       36.3           22.4   
4             월별           01월        586         41             14   
5             월별           02월        326       36.1             21   
6             월별           08월        543       33.2           22.6   
7             월별           09월        358       32.9           20.2   
8             월별           10월        537       42.2           20.5   
9             월별           11월        558       37.7             23   
10            월별           12월        304       32.7           26.9   
11           거주지         서울특별시       1133       37.4           20.1   
12           거주지         부산광역시        251       38.2           16.8   
13           거주지         대구광역시        134       35.8           27.1   
14           거주지         인천광역시        201       37.1           19.6   
15           거주지         광주광역시         91       30.8           21.3   
16           거주지         대전광역시         70       44.4           12.2   
17           거주지         울산광역시         43       25.2             37   
18           거주지         세종자치시         27       29.3           26.6   
19           거주지           경기도        776       40.8           19.9   
20           거주지           강원도         32         17           25.9   
21           거주지           충청도        150       34.8           19.8   
22           거주지           전라도        126       32.2           29.4   
23           거주지           경상도        174         33           22.4   
24           거주지  해외거주 등 기타 지역          5       14.5           25.1   
25       이용 교통수단            항공       3105       37.3           20.8   
26       이용 교통수단            선박        108       31.3             20   
27            성별            남자       1311       37.4           22.1   
28            성별            여자       1902       36.9           19.8   
29            연령        15-20세         56       42.1           21.4   
30            연령        21-30세        704       42.1             17   
31            연령        31-40세       1129         39             21   
32            연령        41-50세        755       31.5           22.2   
33            연령        51-60세        438       33.3             21   
34            연령        61세 이상        130       36.6           28.9   
35         여행 목적     휴가 및 순수여행       2882       38.6           21.4   
36         여행 목적       비즈니스 여행         90       15.5           11.5   
37         여행 목적            기타        241       27.8           16.4   
38          여행형태    개별여행(자유여행)       3213       37.1           20.7   
39          방문횟수            1회        645       36.6           22.3   
40          방문횟수            2회       1070       40.6           19.1   
41          방문횟수            3회        621       38.2             23   
42          방문횟수         4회 이상        878       32.4           19.9   
43  BASE : 문13=1           NaN        NaN        NaN            NaN   

        Unnamed: 5       Unnamed: 6 Unnamed: 7 Unnamed: 8 Unnamed: 9  
0   5만원 이상-10만원 미만  10만원 이상-20만원 미만    20만원 이상       경험없음         평균  
1               10              2.1        0.6       29.5   18535.82  
2              9.5              1.5        0.3       32.9   16250.19  
3             10.1              2.3        0.6       28.2   19441.33  
4              9.5              0.5        0.2       34.9   14034.76  
5              9.6              3.4        0.5       29.5   20227.02  
6             13.9              3.8        0.7       25.9   22999.27  
7              9.1              1.1        0.9       35.9   17363.84  
8              8.2              1.7        0.6       26.8   17255.83  
9              8.3              2.2        0.2       28.5   17364.24  
10            11.5              2.

In [ ]:
columns = {'table' : '분석 카테고리',
           'Unnamed: 1' : '세부 카테고리',
           'Unnamed: 2' : '사례수',
           'Unnamed: 3' : '2만원 미만',
           'Unnamed: 4' : '2만원 이상-5만원 미만',
           'Unnamed: 5' : '5만원 이상-10만원 미만',
           'Unnamed: 6' : '10만원 이상-20만원 미만',
           'Unnamed: 7' : '20만원 이상',
           'Unnamed: 8' : '경험없음',
           'Unnamed: 9' : '평균'}

culture_cost_2020 = culture_cost_2020.rename(columns = columns)
culture_cost_2020

분석 카테고리       세부 카테고리   사례수  2만원 미만  2만원 이상-5만원 미만  5만원 이상-10만원 미만  \
0            NaN           NaN   사례수  2만원 미만  2만원 이상-5만원 미만  5만원 이상-10만원 미만   
1           [전체]           NaN  3213    37.1           20.7              10   
2             구분        01-02월   912    39.2           16.5             9.5   
3             구분        08-12월  2301    36.3           22.4            10.1   
4             월별           01월   586      41             14             9.5   
5             월별           02월   326    36.1             21             9.6   
6             월별           08월   543    33.2           22.6            13.9   
7             월별           09월   358    32.9           20.2             9.1   
8             월별           10월   537    42.2           20.5             8.2   
9             월별           11월   558    37.7             23             8.3   
10            월별           12월   304    32.7           26.9            11.5   
11           거주지         서울특별시  1133    37.4           20.1             9.3   
12           거주지         부산광역시   251    38.2           16.8             9.4   
13           거주지         대구광역시   134    35.8           27.1             8.8   
14           거주지         인천광역시   201    37.1           19.6             5.6   
15           거주지         광주광역시    91    30.8           21.3             9.5   
16           거주지         대전광역시    70    44.4           12.2             8.4   
17           거주지         울산광역시    43    25.2             37            12.8   
18           거주지         세종자치시    27    29.3           26.6            13.1   
19           거주지           경기도   776    40.8           19.9            11.8   
20           거주지           강원도    32      17           25.9            17.6   
21           거주지           충청도   150    34.8           19.8             8.7   
22           거주지           전라도   126    32.2           29.4              10   
23           거주지           경상도   174      33           22.4            12.6   
24           거주지  해외거주 등 기타 지역     5    14.5           25.1               0   
25       이용 교통수단            항공  3105    37.3           20.8             9.8   
26       이용 교통수단            선박   108    31.3             20            15.1   
27            성별            남자  1311    37.4           22.1             9.6   
28            성별            여자  1902    36.9           19.8            10.2   
29            연령        15-20세    56    42.1           21.4             4.1   
30            연령        21-30세   704    42.1             17             7.7   
31            연령        31-40세  1129      39             21            10.4   
32            연령        41-50세   755    31.5           22.2            11.4   
33            연령        51-60세   438    33.3             21            11.4   
34            연령        61세 이상   130    36.6           28.9               8   
35         여행 목적     휴가 및 순수여행  2882    38.6           21.4            10.4   
36         여행 목적       비즈니스 여행    90    15.5           11.5             3.8   
37         여행 목적            기타   241    27.8           16.4             7.5   
38          여행형태    개별여행(자유여행)  3213    37.1           20.7              10   
39          방문횟수            1회   645    36.6           22.3            10.5   
40          방문횟수            2회  1070    40.6           19.1            10.3   
41          방문횟수            3회   621    38.2             23             8.1   
42          방문횟수         4회 이상   878    32.4           19.9            10.5   
43  BASE : 문13=1           NaN   NaN     NaN            NaN             NaN   

    10만원 이상-20만원 미만  20만원 이상  경험없음        평균  
0   10만원 이상-20만원 미만  20만원 이상  경험없음        평균  
1               2.1      0.6  29.5  18535.82  
2               1.5      0.3  32.9  16250.19  
3               2.3      0.6  28.2  19441.33  
4               0.5      0.2  34.9  14034.76  
5               3.4      0.5  29.5  20227.02  
6               3.8      0.7  25.9  22999.27  
7               1.1      0.9  35.9  17363.84  
8               1.7      0.6  

In [ ]:
culture_cost_2020 = culture_cost_2020.drop(index = [0, 1]).reset_index(drop=True)

culture_cost_2020

분석 카테고리       세부 카테고리   사례수 2만원 미만 2만원 이상-5만원 미만 5만원 이상-10만원 미만  \
0             구분        01-02월   912   39.2          16.5            9.5   
1             구분        08-12월  2301   36.3          22.4           10.1   
2             월별           01월   586     41            14            9.5   
3             월별           02월   326   36.1            21            9.6   
4             월별           08월   543   33.2          22.6           13.9   
5             월별           09월   358   32.9          20.2            9.1   
6             월별           10월   537   42.2          20.5            8.2   
7             월별           11월   558   37.7            23            8.3   
8             월별           12월   304   32.7          26.9           11.5   
9            거주지         서울특별시  1133   37.4          20.1            9.3   
10           거주지         부산광역시   251   38.2          16.8            9.4   
11           거주지         대구광역시   134   35.8          27.1            8.8   
12           거주지         인천광역시   201   37.1          19.6            5.6   
13           거주지         광주광역시    91   30.8          21.3            9.5   
14           거주지         대전광역시    70   44.4          12.2            8.4   
15           거주지         울산광역시    43   25.2            37           12.8   
16           거주지         세종자치시    27   29.3          26.6           13.1   
17           거주지           경기도   776   40.8          19.9           11.8   
18           거주지           강원도    32     17          25.9           17.6   
19           거주지           충청도   150   34.8          19.8            8.7   
20           거주지           전라도   126   32.2          29.4             10   
21           거주지           경상도   174     33          22.4           12.6   
22           거주지  해외거주 등 기타 지역     5   14.5          25.1              0   
23       이용 교통수단            항공  3105   37.3          20.8            9.8   
24       이용 교통수단            선박   108   31.3            20           15.1   
25            성별            남자  1311   37.4          22.1            9.6   
26            성별            여자  1902   36.9          19.8           10.2   
27            연령        15-20세    56   42.1          21.4            4.1   
28            연령        21-30세   704   42.1            17            7.7   
29            연령        31-40세  1129     39            21           10.4   
30            연령        41-50세   755   31.5          22.2           11.4   
31            연령        51-60세   438   33.3            21           11.4   
32            연령        61세 이상   130   36.6          28.9              8   
33         여행 목적     휴가 및 순수여행  2882   38.6          21.4           10.4   
34         여행 목적       비즈니스 여행    90   15.5          11.5            3.8   
35         여행 목적            기타   241   27.8          16.4            7.5   
36          여행형태    개별여행(자유여행)  3213   37.1          20.7             10   
37          방문횟수            1회   645   36.6          22.3           10.5   
38          방문횟수            2회  1070   40.6          19.1           10.3   
39          방문횟수            3회   621   38.2            23            8.1   
40          방문횟수         4회 이상   878   32.4          19.9           10.5   
41  BASE : 문13=1           NaN   NaN    NaN           NaN            NaN   

   10만원 이상-20만원 미만 20만원 이상  경험없음        평균  
0              1.5     0.3  32.9  16250.19  
1              2.3     0.6  28.2  19441.33  
2              0.5     0.2  34.9  14034.76  
3              3.4     0.5  29.5  20227.02  
4              3.8     0.7  25.9  22999.27  
5              1.1     0.9  35.9  17363.84  
6              1.7     0.6  26.8  17255.83  
7              2.2     0.2  28.5  17364.24  
8              2.6     1.1  25.2  23207.66  
9              1.8     0.6  30.9  17412.97  
10             2.5     1.2  31.9  18290.12  
11             2.7     1.4  24.2  22695.03  
12             1.9       0  35.8  14815.26  
13             5.5       0  32.7  19739.18  
14               0     3.9    31  21996.72  
15               0  

1) 월별 패턴:

- 1월은 2만원 미만과 2만원 이상-5만원 미만의 범주가 높고, 8월은 5만원 이상-10만원 미만의 범주가 가장 높음.

2) 평균값:

- "08-12월"에는 더 높은 평균 소비액이 있음.

In [ ]:
culture_cost_2020 = culture_cost_2020.drop(index = range(9, 42)).reset_index(drop=True)

culture_cost_2020

분석 카테고리 세부 카테고리   사례수 2만원 미만 2만원 이상-5만원 미만 5만원 이상-10만원 미만 10만원 이상-20만원 미만  \
0      구분  01-02월   912   39.2          16.5            9.5             1.5   
1      구분  08-12월  2301   36.3          22.4           10.1             2.3   
2      월별     01월   586     41            14            9.5             0.5   
3      월별     02월   326   36.1            21            9.6             3.4   
4      월별     08월   543   33.2          22.6           13.9             3.8   
5      월별     09월   358   32.9          20.2            9.1             1.1   
6      월별     10월   537   42.2          20.5            8.2             1.7   
7      월별     11월   558   37.7            23            8.3             2.2   
8      월별     12월   304   32.7          26.9           11.5             2.6   

  20만원 이상  경험없음        평균  
0     0.3  32.9  16250.19  
1     0.6  28.2  19441.33  
2     0.2  34.9  14034.76  
3     0.5  29.5  20227.02  
4     0.7  25.9  22999.27  
5     0.9  35.9  17363.84  
6     0.6  26.8  17255.83  
7     0.2  28.5  17364.24  
8     1.1  25.2  23207.66